training/distributed_training/pytorch/model_parallel_v2/shared-scripts/arguments.py

```diff
@@ -27,7 +27,7 @@ def parse_args():  # pylint: disable=too-many-statements
     opt_grp.add_argument("--seed", type=int, default=12345)
     opt_grp.add_argument("--same_seed", type=int, default=0)
     opt_grp.add_argument("--bf16", default=1, type=int, help="automatic mixed precision training")
-    opt_grp.add_argument("--fp8", default=1, type=int, help="fp8 mixed precision training")
+    opt_grp.add_argument("--fp8", default=0, type=int, help="fp8 mixed precision training")
```

Comment:
Did you run into any issues with the default enabled? Normally, fp8 will only be enabled for models that support fp8 and should switch to bf16 for other cases.